src/services/app/app.ts

```diff
@@ -59,3 +67,18 @@ export const getApps = async (ctx: Context): Promise<App[]> => {
 
   return z.array(App).parse(json);
 };
+
+export const getModels = async (ctx: Context): Promise<ModelApiIdentifier[] | []> => {
+  const cookie = loadCookie();
+  if (!cookie) {
+    return [];
+  }
+
+  assert(ctx.user, "must get user before getting models");
+
+  const edit = new Edit(ctx);
+
+  const result = await edit.query({ query: GADGET_META_MODELS_QUERY, overrides: { endpoint: "/api/graphql" } });
```

Comment:
yah again; I think this is a little strange to use the `edit` client to call the apps meta api

@scott-rc any opinions on how'd you like to do this

Reply:
Yeah, we already separated the underlying [`Client`](https://github.com/gadget-inc/ggt/blob/main/src/services/app/edit/client.ts) from [`Edit`](https://github.com/gadget-inc/ggt/blob/main/src/services/app/edit/edit.ts).

Maybe we could
- Change `Client` to take an endpoint in it's constructor, rather than it building it itself
- Change `Edit` to pass the current `/edit/api/graphql` endpoint to its `Client`
- Create `Api` which is basically a copy paste of `Edit` but passes the `/api/graphql` endpoint to its `Client`

Make sense?